my-config.top

```diff
@@ -9,6 +9,7 @@ base:
     - my-config.personal-pub
     - my-config.untrusted
     - my-config.vault
+    - my-config.dom0-powertop 
```

Comment:
Can you rename this file to `powertop.sls`? It's better to use generic names since this state can be applied to other targets (see [this issue](https://github.com/SkypLabs/vim-config/issues/1) for example).